locales/en.json

```diff
@@ -400,6 +400,7 @@
       "github_cards": "GitHub Cards",
       "grayscale_mode": "Grayscale mode",
       "hide_account_hover_card": "Hide account hover card",
+      "hide_alt_indi_on_posts": "Hide alt indicator on posts",
```

Comment:
```suggestion
      "hide_alt_indicator_on_posts": "Hide alt indicator on posts",
```
?

Reply:
Fair enough. I'll package this with a future pull request if that's ok